Distiller-server/src/main/java/com/mapr/distiller/server/recordtypes/ThreadResourceRecord.java

```diff
@@ -84,78 +127,43 @@ public static boolean produceRecord(RecordQueue thread_resources, String produce
 	public String toString(){
 		return super.toString() + " thread.resources: " + pid + " " + comm + " " + state + " " + ppid;
 	}
-}
-/**
-	public static ThreadResourceRecord diff(ThreadResourceRecord oldRecord, ThreadResourceRecord newRecord){
```

Comment:
I think we misunderstood.  The commented block of code is already being deleted in this checkin.